tests/resources/conformance_suites/edinet/index.xml

```diff
@@ -7,5 +7,6 @@
     <testcase uri="EC0132E/index.xml" />
     <testcase uri="EC0188E/index.xml" />
     <testcase uri="EC0198E/index.xml" />
+    <testcase uri="EC5700W/index.xml" />
```

Comment:
This should be named after GFM because all of the GFM validations have the same `EC5700W` categorization.